packages/content-editor-plugin/src/components/Toolbar/InsertLink.tsx

```diff
@@ -78,7 +73,7 @@ export const InsertLinkDialog = () => {
         const selection = $getSelection();
         if ($isRangeSelection(selection)) {
           const textContent = selection.getTextContent();
-          setValues(values => ({ ...values, text: textContent }));
+          setValues(currentValues => ({ ...currentValues, text: textContent }));
```

Comment:
currentValues isn't a thing here - should update`values` on line 55 where int with useState

Reply:
This is a functional `setState`.  `currentValues` is the previous state.

See --> https://react.dev/reference/react/useState#setstate

Reply:
I will rename the variable to make it clearer

Reply:
Updated